application/views/templates/admin/template.php

```diff
@@ -13,14 +13,28 @@
 		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
 		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
 
-    	<script src="//code.jquery.com/jquery-3.2.1.min.js"></script>
+    <script src="//code.jquery.com/jquery-3.2.1.min.js"></script>
 		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
 
 		<link rel="stylesheet" type="text/css" href="<?php echo base_url(); ?>assets/go/css/go_normalize.css">		
 		<link rel="stylesheet" type="text/css" href="<?php echo base_url(); ?>assets/go/css/go_default.css">	
+		
+		
+		<script type="text/javascript">
```

Comment:
I have adjust the indentation according to you. 
And initially I had write code to merge both onLoad functions in one but footer height was not working. I have merged both onLoad functions code in one. Please check yourselft.
![aftermergeonload](https://user-images.githubusercontent.com/10459685/29202658-faa98c4e-7e87-11e7-8c71-ff2376e330b8.png)
![mergetwoonload](https://user-images.githubusercontent.com/10459685/29202657-faa8a81a-7e87-11e7-8f8e-1ded8affef07.png)